packer/scylla.json

```diff
@@ -168,6 +168,7 @@
       },
       "vm_size": "{{user `vm_size`}}",
       "build_resource_group_name": "scylla-images",
+      "async_resourcegroup_delete": false,
```

Comment:
You probably meant to set it `true`
To be on the safe side - please add a calendar/slack alert  for two months from now to check that we have no left overs resources on Azure caused by this change